javascript/code-challenges/challenges-06.test.js

```diff
@@ -0,0 +1,181 @@
+'use strict';
+
+/* ------------------------------------------------------------------------------------------------
+CHALLENGE 1 - Review
+
+Write a function named getNames that, given an array of people objects, uses map to return an array of names reversed.
+
+For example:
+[
+{
+  name: 'lloyd',
+  age: 32,
+  shoeSize: 12
+},
+{
+  name: 'jamie',
+  age: 21,
+  shoeSize: 8
+}
+]
+
+Returns: ['dyoll', 'eimaj'];
+------------------------------------------------------------------------------------------------ */
+
+const getNames = (arr) => {
+  let arr1 = arr.map(str => str.name);
+  let arr2 = arr1.map(str => str.split("").reverse().join(""));
+ return arr2;
```

Comment:
Great work on the challenges!

Feedback: Can you try combining lines 26 and 27? so that you have 1 map() ---> `str.name.split(' ').reverse().join(' ');` ? Give it a whirl if you have time 👍